specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json

```diff
@@ -114,6 +114,12 @@
           {
             "$ref": "DiskMigrationJobs.json#/parameters/TargetShareParameter"
           },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/TargetScaleUnitParameter"
```

Comment:
is this a bug fix? Can you please make another PR to master branch as well?

Reply:
We have implemented this parameters for the API but did not add them in sepc, as the previous SDK does not need these parameters.
@bganapa If you are going to merge stackadmin to master, do I still need to create a new PR?